client/src/main/java/io/spine/client/QueryBuilder.java

```diff
@@ -41,6 +49,8 @@
  *                                  .where(eq("type", "permanent"),
  *                                         eq("discountPercent", 10),
  *                                         eq("companySize", Company.Size.SMALL))
+ *                                  .orderedBy("name", DESCENDING)
```

Comment:
How come this is `orderedBy`, when we use `SELECT`, `WHERE` and `LIMIT` wording? This should be `orderBy` as well.